apps/client/src/app/implementation-guide-wrapper/custom-menu/custom-menu.component.html

```diff
@@ -4,12 +4,12 @@
     <app-tooltip-icon tooltipKey="ig.customMenu"></app-tooltip-icon>
 
     <div class="pull-right">
-      <button type="button" class="btn btn-primary" title="Generate the XML menu" (click)="generate()">
+      <button type="button" class="btn btn-primary" title="Generate the XML menu" (click)="generate()" [disabled]="readOnly">
```

Comment:
The changes discussed earlier are _only_ for the purposes of the the publish template settings... Doesn't have anything to do with the `custom-menu` component or the `ignore-warnings` component. I think these changes to these two components can be reverted.